glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/emccode/rexray
 import:
   - package: github.com/emccode/libstorage
-    ref:     v0.1.0-rc2
+    ref:     8d0aaaec8cc523697e6ee8de80a22585a4ab58e1
```

Comment:
Hi @clintonskitson,

Please omit the ref altogether for now, at least until we tag LS RC 0.1.0-RC3. That way this build can pull in a new LS commit if we patch it.